raiden-cli/raiden

```diff
@@ -1,6 +1,7 @@
-#!/bin/bash
+#!/bin/sh
 
-RAIDEN="$( dirname $0 )/build/index.js"
-[ ! -e "$RAIDEN" ] && RAIDEN="$( dirname $0 )/build/raiden.bundle.js"
+RAIDEN="$( dirname $0 )/index.js"
+[ ! -e "$RAIDEN" ] && RAIDEN="$( dirname $0 )/build/index.js"
```

Comment:
Why do we need both here?

Reply:
The first/default case is intended for when the script goes in the same directory of one of the output dirs (notice I now copy the script to the bundle, in order to ease launching the bundled cli).
The other 2 cases is to work where the script is, depending on if only the `build` dist is available or only the `bundle` (or both), since you can build only one of them and run the cli from there.

Reply:
Thanks for the explanation.